### PR TITLE
chore: release v1.8.0

### PR DESCRIPTION
**Pull request by tenthirtyam**
### Summary

Updates `CHANGELOG.md`.

### Type

- [ ] Bugfix
- [ ] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [ ] Refactoring
- [x] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### Test and Documentation

- [x] Tests have been completed.
- [x] Documentation has been added or updated.

### CODE REVIEW

File: VMware.CloudFoundation.PasswordManagement.psd1 [hunk 1]
```diff
@@ -11,7 +11,7 @@
     RootModule        = '.\VMware.CloudFoundation.PasswordManagement.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '1.7.2.1008'
+    ModuleVersion     = '1.8.0.1008'
```
Comment (sowjuec) [inline VMware.CloudFoundation.PasswordManagement.psd1]: Shouldn't this be '1.8.0.1000' instead of '1.8.0.1008' ?

**Review by sowjuec — COMMENTED** [review]
The develop branch doesn't have my recent fixes. Please cross check before releasing

File: VMware.CloudFoundation.PasswordManagement.psd1 (hunk 1 above)
Comment (tenthirtyam) [reply VMware.CloudFoundation.PasswordManagement.psd1]: No, decided to keep the increments.

**Comment by tenthirtyam** [comment]
I'm seeing the same for the test but not the JSOM. I can fix that but it was likely a rebase error when correcting the signing.

**Pull request merged**